packages/mattermost-redux/src/client/client4.ts

```diff
@@ -467,6 +469,10 @@ export default class Client4 {
             headers[HEADER_USER_AGENT] = this.userAgent;
         }
 
+        if (options.body) {
+            headers['Content-Type'] = 'application/json';
```

Comment:
should we add `Content-Type` as a header constant similar to the other headers (e.g. `HEADER_USER_AGENT`) ?